package.json

```diff
@@ -17,7 +17,7 @@
         "clean-modules": "rimraf node_modules/ && npm install",
         "lint": "eslint src/**/*.ts",
         "format": "eslint src/**/*.ts --fix",
-        "build:dev": "tsup && tsup --dts-only --outDir dist",
+        "dev": "tsup && tsup --dts-only --outDir dist",
```

Comment:
also you can ignore the change, i committed it by mistake, personal preference of having `dev` lol, i'll revert it here soon